CONTRIBUTING_CODE.md

```diff
@@ -521,10 +521,11 @@ Additionally, Great Expectations adds the following labels to indicate issue sta
 - `good first issue`: identifies issues that provide an introduction to the Great Expectations contribution process
 
 We also have labels to indicate the level of support you can expect for each issue. They are as follows:
-- `gx-supported`: related to a part of the code-base that is tested and actively maintained with new GX Core or GX Cloud releases
 - `community-supported`:  related to a part of the code-base that is not tested and actively maintained with new GX Core or GX Cloud releases; however, we actively welcome ongoing maintenance from the community
 - `not-supported`: an issue that we at GX will not be maintaining, and we will not support PRs or contributions from the community on the topic
 
+Issues without either a `community-supported` or `not-supported` label can be assumed to be **GX-supported**, which means they are related to a part of the code-base that is tested and actively maintained with new GX Core or GX Cloud releases.
```

Comment:
This is likely already in your plans but just in case it's not, I suggest removing the old `gx-supported` label from GitHub to reduce the risk of confusion about whether or not that label is used. 

<img width="1308" alt="Screenshot 2024-10-22 at 11 50 58 AM" src="https://github.com/user-attachments/assets/792fcf2f-6a98-4418-b637-ab61ae588d31">

Reply:
Good thinking. I'll do that now. :)